### PR TITLE
Added cEUR to docs integration checklist

### LINKED ISSUE #7145: Update Developer Guide > Integration > Checklist to also reference cEUR
Update [Developer Guide > Integration > Checklist](https://docs.celo.org/developer-guide/integrations/checklist) to also reference cEUR.

This section of the docs contains references to cUSD. When cEUR is launched, it should be mentioned in these same places.

### LINKED ISSUE #7146: Update Developer Guide > Integration > Listings to also reference cEUR
Make this page more generic such that we have to minimally change it for future stablecoins: https://docs.celo.org/developer-guide/integrations/listings

### PR DESCRIPTION
**Pull request by martinvol**
### Description

Added reference to cEUR

### Related issues

- Fixes #7145

### CODE REVIEW

File: packages/docs/developer-resources/integrations/checklist.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ Celo has [WIP QR code standard CIP16](https://github.com/celo-org/celo-proposals
 Please read more under [Custody](./custody.md), but here is a shortened version:
 
 ### Detect Transfers
-Stable-value currencies, currently just cUSD, is a contract `StableToken` that can be accessed via the ERC20 interface. The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+Stable-value currencies, currently cUSD and cEUR, are contracts, `StableToken` and `StableToken` respectively, that can be accessed via the ERC20 interface.  The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
```
Comment (albertclabs) [inline packages/docs/developer-resources/integrations/checklist.md]: should we say instead:
Stable-value currencies, cUSD and cEUR, are `StableToken` contracts that can be accessed via the ERC20 interface.

Comment (martinvol) [inline packages/docs/developer-resources/integrations/checklist.md]:
```suggestion
Stable-value currencies, currently cUSD and cEUR, are contracts, `StableToken` and `StableTokenEUR` respectively, that can be accessed via the ERC20 interface.  The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
```

Comment (martinvol) [reply packages/docs/developer-resources/integrations/checklist.md]: Thanks!

File: packages/docs/developer-resources/integrations/checklist.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ Celo has [WIP QR code standard CIP16](https://github.com/celo-org/celo-proposals
 Please read more under [Custody](./custody.md), but here is a shortened version:
 
 ### Detect Transfers
-Stable-value currencies, currently just cUSD, is a contract `StableToken` that can be accessed via the ERC20 interface. The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+Stable-value currencies, currently cUSD and cEUR, are contracts, `StableToken` and `StableTokenEUR` respectively, that can be accessed via the ERC20 interface.  The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+
+Addresses for those contracts can be found by querying the [registry](../contractkit/contracts-wrappers-registry.md) or in the (Listing Guide)[`./listings.md`].
```
Comment (martinvol) [inline packages/docs/developer-resources/integrations/checklist.md]:
```suggestion
Addresses for those contracts can be found by querying the [registry](../contractkit/contracts-wrappers-registry.md) or in the [Listing Guide](`./listings.md`).
```

File: packages/docs/developer-resources/integrations/checklist.md [hunk 3]
```diff
@@ -20,7 +20,9 @@ Celo has [WIP QR code standard CIP16](https://github.com/celo-org/celo-proposals
 Please read more under [Custody](./custody.md), but here is a shortened version:
 
 ### Detect Transfers
-Stable-value currencies, currently just cUSD, is a contract `StableToken` that can be accessed via the ERC20 interface. The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+Stable-value currencies, currently cUSD and cEUR, are contracts, `StableToken` and `StableTokenEUR` respectively, that can be accessed via the ERC20 interface.  The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+
+Addresses for those contracts can be found by querying the [registry](../contractkit/contracts-wrappers-registry.md) or in the [Listing Guide](`./listings.md`).
```
Comment (pedro-clabs) [inline packages/docs/developer-resources/integrations/checklist.md]: It seems that the listings link is not working.

**Review by pedro-clabs — APPROVED** [review]
Other than the potentially broken link, looks good to me.

File: packages/docs/developer-resources/integrations/checklist.md (hunk 3 above)
Comment (martinvol) [inline packages/docs/developer-resources/integrations/checklist.md]:
```suggestion
Addresses for those contracts can be found by querying the [registry](../contractkit/contracts-wrappers-registry.md) or in the [Listing Guide](`listings.md`).
```

File: packages/docs/developer-resources/integrations/checklist.md [hunk 4]
```diff
@@ -20,7 +20,9 @@ Celo has [WIP QR code standard CIP16](https://github.com/celo-org/celo-proposals
 Please read more under [Custody](./custody.md), but here is a shortened version:
 
 ### Detect Transfers
-Stable-value currencies, currently just cUSD, is a contract `StableToken` that can be accessed via the ERC20 interface. The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+Stable-value currencies, currently cUSD and cEUR, are contracts, `StableToken` and `StableTokenEUR` respectively, that can be accessed via the ERC20 interface.  The native asset CELO can be accessed via the `GoldToken` ERC20 interface, or natively, similar to ETH on Ethereum.
+
+Addresses for those contracts can be found by querying the [registry](../contractkit/contracts-wrappers-registry.md) or in the [Listing Guide](integrations/listings.md).
```
Comment (martinvol) [inline packages/docs/developer-resources/integrations/checklist.md]:
```suggestion
Addresses for those contracts can be found by querying the [registry](../contractkit/contracts-wrappers-registry.md) or in the [Listing Guide](./listings.md).
```

**Pull request merged**